supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -756,7 +756,7 @@ image repository contains one or more tagged images.
 [discrete]
 [[container-based]]
 ==== image:images/yes.png[yes] container-based (adjective)
-*Description*: Use "container-based" as an adjective when referring to applications made up of multiple services that are distributed in containers. "Container-based" can be used interchangeably with "containerized".
+*Description*: Use _container-based_ as an adjective when referring to applications made up of multiple services that are distributed in containers. "Container-based" can be used interchangeably with "containerized".
```

Comment:
This should stay in quotes since it's giving direction on how to use the term, not defining it.

Reply:
:white_check_mark: